modules/nodes-cluster-resource-override.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * logging/nodes-cluster-overcommit.adoc
+
+[id="nodes-cluster-resource-override_{context}"]
+= Setting cluster-level overcommit using Cluster Resource Override 
+
+The Cluster Resource Override Operator is an admission webhook that allows you to control the level of overcommit and manage
+container density across all the nodes in your cluster by allowing containers on nodes in specific projects to exceed memory and CPU limits.
+
+To use the Cluster Resource Override Operator, create a `ClusterResourceOverride` custom resource (CR), as shown in the
+following example:
+
+----
+apiVersion: operator.autoscaling.openshift.io/v1
+kind: ClusterResourceOverride
+metadata:
+-   name: cluster
+spec:
+   memoryRequestToLimitPercent: 25 <1>
+   cpuRequestToLimitPercent: 25 <2>
+   limitCPUToMemoryPercent: 200 <3>
+----
+
+<1> Optional. If a container memory limit has been specified or defaulted, the memory request is overridden to this percentage of the limit, between 1-100.
+<2> Optional. If a container CPU limit has been specified or defaulted, the CPU request is overridden to this percentage of the limit, between 1-100.
+<3> Optional. If a container memory limit has been specified or defaulted, the CPU limit is overridden to a percentage of the memory limit, with a 100 percentage scaling 1Gi of RAM to equal 1 CPU core. This is processed prior to overriding CPU request (if configured).
+
+[NOTE]
+====
+The Cluster Resource Override Operator overrides have no effect if limits have not
+been set on containers. Create a LimitRange object with default limits per individual project 
+or configure limits in Pod specs in order to for the overrides to apply.
+====
+
+When configured, overrides can be enabled per-project by applying the following
+label to the namespace object for each project: 
+
+----
+clusterresourceoverrides.admission.autoscaling.openshift.io/enabled
+----
+
+The operator watches for the `ClusterResourceOverride` CR and ensures that the `ClusterResourceOverride` admission webhook is installed into the same namespace as the operator. You can check the current state of the admission webhook by checking the status of the cluster custom resource.
+
+----
+$ oc get clusterresourceoverride cluster -o yaml
```

Comment:
Do we need to provide this level details esp the `status` block?

Reply:
Removed parts of output